minecraft-airplane-purpur/minecraft-airplane-purpur.json

```diff
@@ -50,13 +50,6 @@
       "type": "string",
       "userEdit": true
     },
-    "version": {
```

Comment:
I would prefer to keep the variable and update it. Let's hope they use the same filename for 1.18.

Reply:
They won't update to 1.18, but there is a fork of airplane for 1.18 (https://github.com/pufferfish-gg/Pufferfish / https://ci.pufferfish.host), but I think creating a template for that is very difficult since the names of the files are very different from 1.17 and 1.18 on their ci...